src/integrationTest/java/org/opensearch/security/rest/AuthZinRestLayerTests.java

```diff
@@ -156,20 +149,16 @@ public void testShouldPassWithRequiredPermissions() {
 
     @Test
     public void testShouldFailForPOST() {
```

Comment:
testShouldFailForPOST, instead of writing what this test, why are we testing this or what wouldn't work if this behavior was broken, what do you think?

Reply:
This test verifies that user with access to GET endpoint cannot access a POST endpoint. I'll rename it.